### PR TITLE
Adds a DEP feature note regarding vSphere and VW versions

### PR DESCRIPTION
**Pull request by stevsmit**
No BZ.

Preview: https://deploy-preview-36742--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-9-release-notes?utm_source=github&utm_campaign=bot_dp#ocp-4-9-deprecated-removed-features

GH issue at https://github.com/openshift/openshift-docs/issues/33497 (last entry on page). 

Original content by Cody Hoag at https://github.com/openshift/openshift-docs/pull/35530

QE unlikely needed, but maybe?

### CODE REVIEW

**Comment by codyhoag** [comment]
@stevsmit thanks for taking this on! Do you mind adding "vSphere on HW version 13" to the deprecated table as well? https://deploy-preview-36742--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-9-release-notes?utm_source=github&utm_campaign=bot_dp#ocp-4-9-deprecated-removed-features

Probably best we get a QE ack: @jinyunma can you verify this release notes update?

**Comment by stevsmit** [comment]
@codyhoag how's that?

**Comment by codyhoag** [comment]
@stevsmit looks great! The only nit I have is changing both `OpenShift Container Platform` -> `{product-title}.

**Comment by stevsmit** [comment]
> @stevsmit looks great! The only nit I have is changing both `OpenShift Container Platform` -> `{product-title}`.

doh. Fixed.

**Comment by jinyunma** [comment]
LGTM, also cc @duanwei33 @gnufied for reviewing this part.

**Comment by stevsmit** [comment]
@duanwei33 @gnufied please review :)

File: release_notes/ocp-4-9-release-notes.adoc [hunk 1]
```diff
@@ -620,11 +620,20 @@ In the table, features are marked with the following statuses:
...
+|GA
+|DEP
+
 |====
 
 [id="ocp-4-9-deprecated-features"]
 === Deprecated features
 
+[id="ocp-4-9-vsphere-and-virtual-hardware-version-deprecations"]
+==== vSphere 6.7U2 and earlier cluster installation and virtual hardware version 13 are now deprecated
```
Comment (duanwei33) [inline release_notes/ocp-4-9-release-notes.adoc:634]: Do we need to update the "hardware version 13" to "hardware version earlier than 15"? cc @gnufied

Comment (duanwei33) [reply release_notes/ocp-4-9-release-notes.adoc:634]: Others lgtm

Comment (codyhoag) [reply release_notes/ocp-4-9-release-notes.adoc:634]: @duanwei33 what hardware versions were supported for OCP 4.8? I was told that we are deprecating HW 13; stating that we're deprecating all hardware versions below 15 would imply that we supported multiple versions less than 15.

**Review by jeana-redhat — APPROVED** [review]
merge approval :+1:

**Pull request merged**